File: GAPInsurance/Gap.WepApi/Models/PolicyModel.cs
﻿using Gap.Domain;
using Newtonsoft.Json;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace Gap.WepApi.Models
{
    public class PolicyModel
    {
        public Guid PolicyId { get; set; }
        public Guid? CoverTypeId { get; set; }
        public Guid? RiskTypeId { get; set; }
        public Guid? CustomerId { get; set; }
        public DateTime PolicyIssuer { get; set; }
        public int AmontMonths { get; set; }
        public string PolicyValue { get; set; }
       
        public DateTime DateCreated { get; set; }
        public DateTime? DateModified { get; set; }
        public bool FlagDisable { get; set; }
        public string PolicyName { get; set; }
        public int PolicyNumber { get; set; }
        public TypeRiskModel TypeRisks { get; set; }
        public CoverTypeModel CoverTypePolicy { get; set; }
        public CustomerModel customerPolicies { get; set; }

    }
}

File: GAPInsurance/GapInsuranceClient/GapClientApp/src/app/shared/policy.model.ts
import { BrowserDynamicTestingModule } from '@angular/platform-browser-dynamic/testing';
import { Typerisk } from './typerisk.model';
import { Covertype } from './covertype.model';
import { Customer } from './customer.model';

export class Policy {
    PolicyId:string;
    CoverTypeId:string;
    RiskTypeId:string;
    CustomerId:string;
    PolicyIssuer:Date;
    AmontMonths:number;
    PolicyValue:string;
    FlagDisable:boolean;
    PolicyName:string;
    PolicyNo:string;

}

export class Policies {
    PolicyId:string;
    CoverTypeId:string;
    RiskTypeId:string;
    CustomerId:string;
    PolicyIssuer:Date;
    AmontMonths:number;
    PolicyValue:string;
    FlagDisable:boolean;
    PolicyName:string;
    PolicyNo:string;
    DateCreated: Date;
    DateModified: Date;
    TypeRisks: Typerisk;
    CoverTypePolicy: Covertype;
    customerPolicies: Customer;

}

